### PR TITLE
Small CSS changes across project to match colour scheme and designs

### LINKED ISSUE #344: Small CSS colour fixes across project based on new colour scheme
### What are you proposing?

- Complete a full sweep of the project to fix any colours that don't match the new colour scheme.

### Is this ready to implement?
Yes.

Colour scheme to follow:
Screen Shot 2021-04-17 at 12 39 10 AM

### PR DESCRIPTION
**Pull request by lhuijser**
# Description
This PR closes #344 

Updated the following styles to match the new colour scheme and hi-fi designs:

- Various buttons and their hover-over styles such as the 'create new game', 'join game', and 'leave game' buttons - changed colours
- Lobby game settings panel - changed colours
- Lobby id - changed colour
- Player selected red background - changed colour and added rounded corners
- Shifted settings icon in the home page to the right of the screen

# Demo
Screen Shot 2021-04-19 at 3 56 51 PM
Screen Shot 2021-04-19 at 3 56 37 PM
Screen Shot 2021-04-19 at 3 56 20 PM
Screen Shot 2021-04-19 at 3 56 10 PM

# Type of change:
(Delete the ones that are not relevant)
* UI update

# Checklist:
* Have you merged main into your branch?
* Have you tested your changes to ensure it works as expected and does not break existing functionality?
* If applicable, please ensure sufficient tests are added that is related to the changes.
* Please ensure this PR has a `label`, is linked to an `issue` and is related to a `project`
* If new documentation is required for this change, have you created a new documentation issue that describes the documentation needed?

### CODE REVIEW

**Review by AngelaLH — DISMISSED** [review]
Did a quick run through and all the colours are looking great :D Thank you for all of your changes :) especially good to have consistent red colours

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,38 +1,3 @@
 .App {
     text-align: center;
 }
-
```
Comment (milkyway404) [inline frontend/src/App.css:4]: awesome to delete the default stuff!

File: frontend/src/Components/LobbySettings.jsx [hunk 2]
```diff
@@ -57,24 +57,24 @@ const LobbySettings = ({ setOpen, setOpenInfo }) => {
             <CardContent className={classes.content}>
                 <List>
                     <ListItem>
-                        <Typography style={{ marginRight: '20px' }}>Number of Mafias: </Typography>
-                        <Select style={{ color: '#E3F1F1' }} defaultValue={1} label="This is a label">
+                        <Typography style={{ marginRight: '20px', color: 'black' }}>Number of Mafias: </Typography>
```
Comment (milkyway404) [inline frontend/src/Components/LobbySettings.jsx:60]: nitpick: would it be better to put this in a css?

**Review by milkyway404 — DISMISSED** [review]
I love this!! Especially changing the hover over colours as well. Slight nitpick about using styles in-line but it's not too big.

**Review by Benjamin-Piper — DISMISSED** [review]
LGTM, but it's worth noting that the settings icon in the homepage is on the left, whereas it's on the right elsewhere

File: frontend/src/Components/LobbySettings.jsx (hunk 2 above)
Comment (lhuijser) [reply frontend/src/Components/LobbySettings.jsx:60]: Tried but doesn't appear to be an easy fix, will leave for now

**Comment by lhuijser** [comment]
> LGTM, but it's worth noting that the settings icon in the homepage is on the left, whereas it's on the right elsewhere

Thank you! I have shifted this settings icon to the right on the home page :)

**Review by milkyway404 — DISMISSED** [review]

**Review by Benjamin-Piper — DISMISSED** [review]
Thanks for making those changes! I approve

**Review by Benjamin-Piper — APPROVED** [review]
3rd time's the charm!

**Review by ysway — APPROVED** [review]
LGTM, thank you Laura

**Pull request merged**